src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final int n = 3;
+        for (int i = 0; i < n; i++) {
+            Lottery lottery = new Lottery();
```

Comment:
don't create an object in the for loop if you don't need it, it could get pretty slow in some cases - Create lottery before the loop and reuse the already created object it here